init.rb

```diff
@@ -21,4 +21,5 @@
   end
```

Comment:
I think you should increase the version as well.

Reply:
Ok. I increased it to 1.1.0